pkg/userinfo/roleRef.go

```diff
@@ -127,3 +129,64 @@ func matchUserOrGroup(subject rbacv1.Subject, userInfo authenticationv1.UserInfo
 
 	return false
 }
+
+//IsRoleAuthorize is role authorize or not
+func IsRoleAuthorize(rbLister rbaclister.RoleBindingLister, crbLister rbaclister.ClusterRoleBindingLister, rLister rbaclister.RoleLister, crLister rbaclister.ClusterRoleLister, request *v1beta1.AdmissionRequest) (bool, error) {
+	if strings.Contains(request.UserInfo.Username, SaPrefix) {
+		roles, clusterRoles, err := GetRoleRef(rbLister, crbLister, request)
+		if err != nil {
+			return false, err
+		}
+
+		for _, e := range clusterRoles {
+			if strings.Contains(e, "kyverno:") {
+				return true, nil
+			} else {
+				role, err := crLister.Get(e)
+				if err != nil {
+					return false, err
+				}
+				labels := role.GetLabels()
+
+				if labels["kubernetes.io/bootstrapping"] == "rbac-defaults" {
+					return true, nil
+				}
+			}
+		}
+		for _, e := range roles {
+			roleData := strings.Split(e, ":")
+			role, err := rLister.Roles(roleData[0]).Get(roleData[1])
+			if err != nil {
+				return false, err
+			}
+			labels := role.GetLabels()
+			if !strings.Contains(e, "kyverno:") {
+				if labels["kubernetes.io/bootstrapping"] == "rbac-defaults" {
+					return true, nil
+				}
+			}
+		}
+	} else {
+		// User or Group
+		excludeDevelopmentRole := []string{"minikube-user", "kubernetes-admin"}
```

Comment:
These are the known users we want to exclude, we need to think of how to make it configurable (dynamically) if there are more users who need to be excluded.